project/src/main/world-data.gd

```diff
@@ -0,0 +1,25 @@
+extends Node
```

Comment:
Add comments.

Reply:
Added a class-level comment, `Stores information about all worlds, such as their music, intermissions and levels.`. Most of this code is otherwise self-documenting and doesn't warrant comments.